client/layout/controller.js

```diff
@@ -50,13 +50,24 @@ export const getPages = ( homepageEnabled ) => {
 	const pages = [];
 	const initialBreadcrumbs = [ [ '', wcSettings.woocommerceTranslation ] ];
 
+	const getUrlParams = ( locationSearch ) =>
```

Comment:
Instead of having this code twice, what do you think about extracting it to a util method and adding some tests around it?

Reply:
It's a great idea @timmyc!
As it will be a `util` used in the header and in the layout components, I created a new folder `client/utils` to keep the utils that are used by different pages of the client.
If you think that it should be different just let me know and I will change it.
Also I added some tests 🙂
The changes are in the commit 9e81c5a